x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/use_columns.tsx

```diff
@@ -199,7 +199,7 @@ export const useColumns = (
       'data-test-subj': 'mlAnalyticsTableRowDetailsToggle',
     },
     {
-      name: 'ID',
+      name: 'id',
```

Comment:
Is this what defines the label for the column in the jobs list? The label needs to be uppercase, to be consistent with the anomaly detection jobs list, Overview tables, and wizards.

Reply:
`ID` name was causing a bug because column names should match the actual field for sorting to work. Managed to keep the uppercase notation by setting a `field` prop. Check https://github.com/elastic/kibana/pull/83439/commits/54efa0d781a36cdda54d518f50ce60ef91bcbfca and https://github.com/elastic/kibana/pull/83439/commits/7c4a4c816b8f3d7a3c9998ff210d013184f5e174